tyk-pro/templates/bootstrap-service.yml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.bootstrap -}}
+{{- if .Values.bootstrap.enabled -}}
 apiVersion: v1
 kind: ServiceAccount
```

Comment:
Should we rename this file to bootstrap-serviceaccount?

Reply:
Fixed